base/acme/conf/engine.conf

```diff
@@ -12,11 +12,29 @@ policy.wildcard=true
 policy.nonceValidity.length=30
 policy.nonceValidity.unit=MINUTES
 
+policy.pendingAuthorizationValidity.length=30
+policy.pendingAuthorizationValidity.unit=MINUTES
+
+policy.invalidAuthorizationValidity.length=30
+policy.invalidAuthorizationValidity.unit=MINUTES
+
 policy.validAuthorizationValidity.length=30
 policy.validAuthorizationValidity.unit=MINUTES
 
 policy.pendingOrderValidity.length=30
 policy.pendingOrderValidity.unit=MINUTES
 
+policy.invalidOrderValidity.length=30
+policy.invalidOrderValidity.unit=MINUTES
+
+policy.readyOrderValidity.length=30
+policy.readyOrderValidity.unit=MINUTES
+
+policy.processingOrderValidity.length=30
+policy.processingOrderValidity.unit=MINUTES
+
 policy.validOrderValidity.length=30
 policy.validOrderValidity.unit=MINUTES
+
+policy.certificateValidity.length=30
```

Comment:
To be clear, this isn't about the lifetime ("expiration date" / "Valid Until") on the X509 Certificate, but the length of time the record of the successful certificate issuance is kept in the database, correct?

Perhaps we should rename it "policy.database.*" -- to be clear that it is a database record retention policy, and that actual X509 Certificate's "Valid Until" field is controlled by the CA's issuance.

Reply:
Yes, that's why I was suggesting to rename "validity" into "retention/expiration/purging" earlier to avoid confusions with cert "not later" attribute. If we use "policy.database" how would the full configuration parameters look like? If we use "retention" they would look like this:
```
# Records retention policy
policy.nonceRetention.length=30
policy.nonceRetention.unit=MINUTES

policy.validAuthorizationRetention.length=30
policy.validAuthorizationRetention.unit=MINUTES
```
or maybe:
```
# Records retention policy
policy.retention.nonce.length=30
policy.retention.nonce.unit=MINUTES

policy.retention.validAuthorization.length=30
policy.retention.validAuthorization.unit=MINUTES
```

Reply:
Agreed. Let me update the PR.